config/dev/pulse.json

```diff
@@ -0,0 +1,362 @@
+{
```

Comment:
I added this file to not force us manually removing the queues each time for testing.